src/Cli/Microsoft.TemplateEngine.Cli/TemplatePackageCoordinator.cs

```diff
@@ -472,7 +472,10 @@ internal void DisplayNuGetPackageMetadata(NugetPackageMetadata packageMetadata,
         {
             reporter.WriteLine($"{packageMetadata.Identity.Id}");
             WriteIfNotNull(LocalizableStrings.DetailsCommand_Property_Version, packageMetadata.PackageVersion.ToString(), reporter, 1);
-            WriteIfNotNull(LocalizableStrings.DetailsCommand_Property_PrefixReserved, packageMetadata.PrefixReserved.ToString(), reporter, 1);
+            if (packageMetadata.PrefixReserved != null && packageMetadata.Source.Source.Equals(NugetOrgFeed))
+            {
+                WriteIfNotNull(LocalizableStrings.DetailsCommand_Property_PrefixReserved, packageMetadata.PrefixReserved.ToString(), reporter, 1);
```

Comment:
Could you use GetFormattedValue for displaying "✔" or "✘" instead of true/false?

Reply:
and please rename GetFormattedValue -> GetFormattedBooleanValue

Reply:
This will not be changed as some command line prompts are not able to correctly identify it
<img width="339" alt="image" src="https://github.com/dotnet/sdk/assets/10743736/2179daf9-a13d-47c6-8db4-e7f2be99c5e8">